README.md

```diff
@@ -61,6 +61,59 @@ Table of Contents
 
 ## Things to be aware of when writing code
 
+###Writing secure software
+While not always the most exciting part of writing code, it is important
+to consider the security risks with a software project to avoid costly
+patches and perhaps more importantly, reputation damage. Security risks
+analysis should be employed at the early stage of a project (it is
+usually way more expensive and complicated at a late stage) and
+continuously carried out until the end. 
+
+The security risks will differ vastly between project, but general
+guidelines are:
+
+* Educate yourself, research on what security risks are related to
+ the different components of your project. Some example resources are:
+  * https://bestpractices.coreinfrastructure.org/en
+  * https://owasp.org/www-project-top-ten/
+  * https://owasp.org/www-project-secure-coding-practices-quick-reference-guide/migrated_content
+* Don’t blindly trust out-of-the-box software and default
+ configurations, e.g. popular Docker images, aws services, etc. 
+ Malicious software can be present and default configurations usually
+ have simplicity as the primary goal, not security.
+* Incorporate security in the entire Software Development Lifecycle:
+  * Include security considerations in Requirements Gathering & Analysis
+  * Threat model during Design Stage
+  * Use/Implement tests/build processes/libraries with security in
+   mind during Development Stage
+  * Perform security testing during Testing Stage
```

Comment:
"Testing Stage" and "Design Stage" etc. sounds like you're referring to a formal process. What is that process? If not referring to a formal process, use lower-case.